sensors/sensors_test.go

```diff
@@ -17,14 +17,19 @@ func TestValidateGetData(t *testing.T) {
 	logger := golog.NewTestLogger(t)
 	ctx := context.Background()
 
-	sensors := []string{"good_lidar"}
-	goodLidar, err := s.NewLidar(ctx, s.SetupDeps(sensors), sensors, logger)
+	lidar := map[string]string{"name": "good_lidar"}
```

Comment:
why are we creating a map, when only the value of one of its keys are used... Could we please remove all the cam `lidar` in this test file & change them to just be strings?